frontend/stories/components/application/opportunity.mock.json

```diff
@@ -0,0 +1,63 @@
+{
```

Comment:
we should discuss at some point how we want to handle things like this - I've been defaulting to putting things in .ts files as in https://github.com/HHS/simpler-grants-gov/blob/main/frontend/src/utils/testing/fixtures.ts but we have some stuff that is in individual mocks as json as well (notably this, which I think is out of date and may be unused at this point, but could be updated for use a fixture for this story theoretically - https://github.com/HHS/simpler-grants-gov/blob/main/frontend/src/app/api/mock/APIMockResponse.json). It'd be nice to have a documented preference here